pantos/validatornode/business/transfers.py

```diff
@@ -259,6 +262,33 @@ def get_validator_nonce(self, source_blockchain: Blockchain,
                 source_blockchain=source_blockchain,
                 source_transaction_id=source_transaction_id)
 
+    def submit_transfer_offchain(self, internal_transfer_id: int,
```

Comment:
Why do you need the `internal_transfer_id` and `transfer: CrossChainTransfer` here as parameter? Based on the name of the function, I assume that `transfer` is submitted to the primary validator. Do you plan on doing any database operations here?

I also feel that the naming of the function is not well chosen. I would maybe change it to something like `submit_transfer_to_primary_node` ->  by reading the name it is clear what is happening.
After reading `submit_transfer_offchain` I had to read the docstring to know what is going to happen.

Reply:
`CrossChainTransfer` is not a database entity, it's a cross-layer entity transferred from detection over validation to submission (the step we are talking about here) and then to confirmation business logic tasks.

The steps involved for the secondary node's submission business logic are (see PAN-1981):
- Get the validator nonce from the primary node.
- Update the validator nonce in the database.
- Sign the cross-chain transfer.
- Submit the transfer signature to the primary node.

The name `submit_transfer_offchain` was chosen for secondary nodes because the primary node's submission business logic is implemented in `submit_transfer_onchain`. But I'm also fine with `submit_transfer_to_primary_node`, I'm going to change that.